scripts/scr_add_man/scr_add_man.gml

```diff
@@ -163,29 +138,20 @@ function scr_add_man(man_role, target_company, spawn_exp, spawn_name, corruption
 	// End of Imperials
 		// Faction 6 - Eldar
 			case "Ranger":
-				obj_ini.wep1[target_company][good] = "Ranger Long Rifle";
-				obj_ini.wep2[target_company][good] = "Eldar Power Sword";
-				obj_ini.armour[target_company][good] = "Ranger Armour"; // I should add "Eldar Armour" to the fellow too
-				obj_ini.experience[target_company][good] = 180; // They should have much more EXP (like 250 or more) as they have quite the lifespan
+				spawn_exp = 180; // They should have much more EXP (like 250 or more) as they have quite the lifespan
 				obj_ini.race[target_company][good] = 6
-				unit = new TTRPG_stats("mechanicus", target_company, good, "skitarii_ranger"); // TODO - add aeldari religion in relevant gml files
+				unit = new TTRPG_stats("Eldari", target_company, good, "eldar_ranger"); // TODO - add aeldari religion in relevant gml files
 				break;
 		// Faction 6.5 - Dark Eldar?
 		// Faction 7 - Orks
 			case "Ork Sniper":
-				obj_ini.wep1[target_company][good] = "Sniper Rifle";
-				obj_ini.wep2[target_company][good] = "Choppa";
-				obj_ini.armour[target_company][good] = "Ork Armour";
-				obj_ini.experience[target_company][good] = 20;
-				obj_ini.race[target_company][good] = 7;
+				spawn_exp = 20;
+				obj_ini.race[target_company][good] = eFACTION.Ork;
```

Comment:
A question on my end - in earlier hireling stats, we see `obj_ini.race` use numbers, but here they have an entire `eFACTION.Ork`. It can work with both cases?
Otherwise, it is a bit of a minor complaint that this is inconsistent stat cards, as ones earlier use numbers and this one use phrases. But this can be left for future PRs.